### PR TITLE
Kit: Fix - Empty logo caused a PHP error.

### PR DESCRIPTION
**Pull request by matipojo**

### CODE REVIEW

File: core/kits/documents/tabs/site-identity.php [hunk 1]
```diff
@@ -21,6 +21,7 @@ public function get_title() {
 
 	public function register_tab_controls() {
 		$custom_logo_id = get_theme_mod( 'custom_logo' );
+		$custom_logo_src = wp_get_attachment_image_src( $custom_logo_id, 'full' );
```
Comment (kobizz) [inline core/kits/documents/tabs/site-identity.php:24]: Is there a situation that the id exists, but the src doesn't?

**Review by kobizz — CHANGES_REQUESTED** [review]

File: core/kits/documents/tabs/site-identity.php (hunk 1 above)
Comment (matipojo) [reply core/kits/documents/tabs/site-identity.php:24]: Yes, a delete attachment.

Comment (kobizz) [reply core/kits/documents/tabs/site-identity.php:24]: In this case, you shouldn't print the id at all

Comment (kobizz) [reply core/kits/documents/tabs/site-identity.php:24]: In this case, do you need to print the ID?

Comment (matipojo) [reply core/kits/documents/tabs/site-identity.php:24]: yes, in order the media control will show that a value is exist.

Comment (kobizz) [reply core/kits/documents/tabs/site-identity.php:24]: There are still some conflicts that need to be resolved

Comment (matipojo) [reply core/kits/documents/tabs/site-identity.php]: Done

**Review by kobizz — APPROVED** [review]

**Pull request merged**